<ui version="4.0" >
 <author>Teemu Rytilahti</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Akregator::SettingsGeneral</class>
 <widget class="QWidget" name="Akregator::SettingsGeneral" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>General</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="Q3GroupBox" name="groupBox3_2" >
     <property name="title" >
      <string>Global</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" colspan="2" >
       <widget class="QCheckBox" name="kcfg_UseIntervalFetch" >
        <property name="text" >
         <string>&amp;Use interval fetching</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QCheckBox" name="kcfg_UseNotifications" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="whatsThis" >
         <string>Select this if you want to get notified when there are new articles.</string>
        </property>
        <property name="text" >
         <string>Use &amp;notifications for all feeds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QCheckBox" name="kcfg_ShowTrayIcon" >
        <property name="text" >
         <string>Show tra&amp;y icon</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Fetch feeds every:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="kcfg_AutoFetchInterval" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="specialValueText" >
         <string>1 minute</string>
        </property>
        <property name="suffix" >
         <string> minutes</string>
        </property>
        <property name="maximum" >
         <number>3600</number>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="singleStep" >
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="Q3GroupBox" name="groupBox3" >
     <property name="title" >
      <string>Startup</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="kcfg_MarkAllFeedsReadOnStartup" >
        <property name="text" >
         <string>Mark &amp;all feeds as read on startup</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="kcfg_FetchOnStartup" >
        <property name="text" >
         <string>Fetch all fee&amp;ds on startup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="Q3GroupBox" name="groupBox3_3" >
     <property name="title" >
      <string>Network</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="kcfg_UseHTMLCache" >
        <property name="text" >
         <string>Use the &amp;browser cache (less network traffic)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends></extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
